test/e2e/circle-style.test.js

```diff
@@ -0,0 +1,30 @@
+const { assert, driver } = require('vl-ui-core').Test.Setup;
+const VlMapCircleStylePage = require('./pages/vl-map-circle-style.page');
+
+describe('vl-map-circle-style', async() => {
+    const vlMapPage = new VlMapCircleStylePage(driver);
+
+    before(() => {
+        return vlMapPage.load();
+    });
+
+    it('Als gebruiker kan ik een layer met circle stijl definieren', async () => {
+        const layer = await vlMapPage.getStandaardLayer();
+
+        await assert.eventually.isTrue(layer.hasCircleStyle());
+    });
+
+    it('Als gebruiker kan ik een layer met circle stijl definiëren met aangepaste stijl', async () => {
+        const layer = await vlMapPage.getAangepasteCircleStyleLayer();
+
+        await assert.eventually.isTrue(layer.hasCircleStyle());
+        const style = await layer.getCircleStyle();
```

Comment:
op deze manier moet een layer alle soorten styles kennen, en moet ge ook elke keer de maplayer aanpassen als er nieuwe soort style komt.
dus ik zou toch eens kijken of ge de dependency hier niet komt omdraaien op een of andere manier.